data_science/scripts/create_tunnel_to_asg.py

```diff
@@ -3,35 +3,48 @@
 """
 Connect to the first instance in an autoscaling group.
 
-Usage: create_tunnel_to_asg.py --key=<KEY> [--port=<PORT>]
+Usage: create_tunnel_to_asg.py [--key=<KEY>] [--port=<PORT>] [--type=<INSTANCE_TYPE>]
 
 Actions:
-  --key=<KEY>     Path to an SSH key with access to the instances in the ASG.
-  --port=<PORT>   Local port to use for the remote Jupyter notebook
-                  (default: 8888).
+  --key=<KEY>             Path to an SSH key with access to the instances in the ASG.
+  --port=<PORT>           Local port to use for the remote Jupyter notebook
+                          (default: 8888).
+  --type=<INSTANCE_TYPE>  AWS Instance type (valid values: p2,t2) defaults to t2
```

Comment:
If you modify the help string to be:

```
--type=(p2 | t2)
```

I think docopt will check you only pass one of these values.